### PR TITLE
[WIP] ToolBar Typescript&material-ui

### PR DESCRIPTION
**Pull request by BohanZhai**
In this PR, ToolBar is reconstructed with material-ui react and typescript. The new ToolBar is implemented with configurable attributes.  For example, we gave the screenshot of ToolBar for 2d video labeling tool. As shown below, it has `Occluded `, `Truncated`, and `Traffic Light Color` switch buttons, and `Remove`, `End Object Track`, and `Track-Link` buttons for 2d video label type, which are parsed from a Json file. 

Maybe we are going to slightly change the material-ui component styles in the future, but the functionalities are implemented at this stage.

Demo:
To show our configurable attribute, we modify the attribute field in the json file, and see the difference.
This is the screenshot for the following json file:

demo1-1

demo001

After changing the attribute, the screenshot for the following json file is:
demo6-1
demo6

As we can see, the buttons are changed.

- [x] Reconstruct `ToolBar.tsx` with material-ui 
- [x] Create rendering function for ToolBar in `common/label.tsx`
- [x] Call ToolBar component in `window.tsx` and pass correct parameters to it.
- [x] Set up corresponding settings in `tscofig .js`and `webpack.config.js`

### CODE REVIEW

**Comment by fyu** [comment]
This pull request **introduces 38 alerts** when merging 953b7ddb0e8e7aa51a9ad8d9333b2443f73d1a81 into 16268d2d0f9db0aca8c386d21f95247dd18f47f0 - [view on LGTM.com](https://lgtm.com/projects/g/ucbdrive/scalabel/rev/pr-f7f077de236f2ae08e7a26ed408403f3da255940)

**new alerts:**

* 38 for Syntax error

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by fyu** [comment]
This pull request **introduces 3 alerts** when merging 55488c606d4620723f9ecc8534bf157f1d3d2624 into 16268d2d0f9db0aca8c386d21f95247dd18f47f0 - [view on LGTM.com](https://lgtm.com/projects/g/ucbdrive/scalabel/rev/pr-94ee63885734095d125db5c9b90c2464a77090a6)

**new alerts:**

* 2 for Unused variable, import, function or class
* 1 for Unused or undefined state property

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by fyu** [comment]
This pull request **introduces 3 alerts** when merging bed924f01249bef20ded88f783e78498faa4e8e4 into 3884c117edab006893c29d5e36e2fd76d36b6c5c - [view on LGTM.com](https://lgtm.com/projects/g/ucbdrive/scalabel/rev/pr-bb40a2c522f57b5fe09150ec50a9183e3c434e52)

**new alerts:**

* 2 for Unused variable, import, function or class
* 1 for Unused or undefined state property

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by fyu** [comment]
This pull request **introduces 3 alerts** when merging be07f0fd60a182fd31b5d069099bc6e143bbf931 into 3884c117edab006893c29d5e36e2fd76d36b6c5c - [view on LGTM.com](https://lgtm.com/projects/g/ucbdrive/scalabel/rev/pr-af60a36b47b99b6c161a18eb5b9ac557f5bcdb6b)

**new alerts:**

* 2 for Unused variable, import, function or class
* 1 for Unused or undefined state property

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by fyu** [comment]
This pull request **introduces 1 alert** when merging 55b9956819a4a556f0e69d10620a76bac1a646ce into 3884c117edab006893c29d5e36e2fd76d36b6c5c - [view on LGTM.com](https://lgtm.com/projects/g/ucbdrive/scalabel/rev/pr-1fc1e3aa6e82a5d8cf53963a45126ee38786379e)

**new alerts:**

* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by fyu** [comment]
This pull request **introduces 3 alerts** when merging 76ded57e278f0d860fe96d03658c91e88526a936 into 3884c117edab006893c29d5e36e2fd76d36b6c5c - [view on LGTM.com](https://lgtm.com/projects/g/ucbdrive/scalabel/rev/pr-a2c3b8b2402b310a65d42591ca0cd2d049b52ec6)

**new alerts:**

* 3 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by fyu** [comment]
This pull request **introduces 3 alerts** when merging f037589a975185ffbd18e780a3974209168c0d32 into 3884c117edab006893c29d5e36e2fd76d36b6c5c - [view on LGTM.com](https://lgtm.com/projects/g/ucbdrive/scalabel/rev/pr-3462b862a6d73830b9fb7161c89f5e522a44f419)

**new alerts:**

* 3 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by fyu** [comment]
This pull request **introduces 2 alerts** when merging f4756822209b46918cd11fe2f1a67bf5d67ea5a7 into 3884c117edab006893c29d5e36e2fd76d36b6c5c - [view on LGTM.com](https://lgtm.com/projects/g/ucbdrive/scalabel/rev/pr-1b48755c0c6a7661e5f8d8f12af5b10d38c09e6a)

**new alerts:**

* 2 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by fyu** [comment]
This pull request **introduces 2 alerts** when merging 6b1c60595d4c3055d1c64f9574d2eb7c2b8d879c into 3884c117edab006893c29d5e36e2fd76d36b6c5c - [view on LGTM.com](https://lgtm.com/projects/g/ucbdrive/scalabel/rev/pr-521ffeb5a3ec4bb03882d1824a8bcaf5e7108f1f)

**new alerts:**

* 2 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by fyu** [comment]
This pull request **introduces 2 alerts** when merging 90a017158a877e73d94519c6920843030798df04 into 3884c117edab006893c29d5e36e2fd76d36b6c5c - [view on LGTM.com](https://lgtm.com/projects/g/ucbdrive/scalabel/rev/pr-c09a2559534db90389e26099b3447b071dcb2acc)

**new alerts:**

* 2 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by fyu** [comment]
This pull request **introduces 2 alerts** when merging 829e7136d18862e7fa205b5f1162c29a29aeaeed into 3884c117edab006893c29d5e36e2fd76d36b6c5c - [view on LGTM.com](https://lgtm.com/projects/g/ucbdrive/scalabel/rev/pr-8aee53c7dc93622da8aeb4fa24d3af5f7f5ff91e)

**new alerts:**

* 2 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by fyu** [comment]
This pull request **introduces 1 alert** when merging 53eb47869bf9f8b999bf865e8477b027f5a52cf5 into 3884c117edab006893c29d5e36e2fd76d36b6c5c - [view on LGTM.com](https://lgtm.com/projects/g/ucbdrive/scalabel/rev/pr-dfbefb7649574e8a6201a1f4b4c4ebe05ae23c20)

**new alerts:**

* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by fyu** [comment]
This pull request **introduces 1 alert** when merging b3ab1b6f16d3f8929e2f84a984d2300c33e25aa8 into 3884c117edab006893c29d5e36e2fd76d36b6c5c - [view on LGTM.com](https://lgtm.com/projects/g/ucbdrive/scalabel/rev/pr-c7347563d5284b37cf026d215dd30e00842a195e)

**new alerts:**

* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by fyu** [comment]
This pull request **introduces 2 alerts** when merging 356024ab9043124746d633c87e2134b8e9c6c864 into a0211457f6c67850d69592067a9b360dbb7f33db - [view on LGTM.com](https://lgtm.com/projects/g/ucbdrive/scalabel/rev/pr-1ceb5084a94183c6161204d1b749c4f90a8fca2b)

**new alerts:**

* 1 for Unused variable, import, function or class
* 1 for Unused or undefined state property

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by fyu** [comment]
This pull request **introduces 2 alerts** when merging 27e9c6ac5c27b8bfefaa1701d45f0dda667949fc into a0211457f6c67850d69592067a9b360dbb7f33db - [view on LGTM.com](https://lgtm.com/projects/g/ucbdrive/scalabel/rev/pr-4967b9109cebc35add3e5bd1e4b30f991084decb)

**new alerts:**

* 1 for Unused variable, import, function or class
* 1 for Unused or undefined state property

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by fyu** [comment]
This pull request **introduces 2 alerts** when merging d1561e6c5a5ce0e18266083d28b8962f3fcc1164 into a0211457f6c67850d69592067a9b360dbb7f33db - [view on LGTM.com](https://lgtm.com/projects/g/ucbdrive/scalabel/rev/pr-f189402b318819540bcbcb3c287d99255401e096)

**new alerts:**

* 1 for Unused variable, import, function or class
* 1 for Unused or undefined state property

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by fyu** [comment]
This pull request **introduces 2 alerts** when merging 3f1d83079dd018a28df729e5478cf1111a87f648 into a0211457f6c67850d69592067a9b360dbb7f33db - [view on LGTM.com](https://lgtm.com/projects/g/ucbdrive/scalabel/rev/pr-e396e68c8ccc1176acc99e0f905abcd9575194e8)

**new alerts:**

* 1 for Unused variable, import, function or class
* 1 for Unused or undefined state property

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by fyu** [comment]
This pull request **introduces 4 alerts** when merging 2c814745d6f37309c884972a0836245a31cacee4 into a14928b7bbbea2dbc6851b1a1a522ed30e1acdc3 - [view on LGTM.com](https://lgtm.com/projects/g/ucbdrive/scalabel/rev/pr-ab2d1ceaee9f6d328cc8a7a4502625d0f156f1b5)

**new alerts:**

* 4 for Syntax error

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by yuandong** [comment]
For the traffic light UI
You can use the toggle button here: https://material-ui.com/lab/api/toggle-button/
It's exactly what you'd like to use. 

See demo https://codesandbox.io/s/wqz12o386k?fontsize=14
With some additional simple CSS, you can get what you want

**Comment by fyu** [comment]
This pull request **introduces 1 alert** when merging 66ad145fed6ac22cde354db90c652e191fc19888 into a14928b7bbbea2dbc6851b1a1a522ed30e1acdc3 - [view on LGTM.com](https://lgtm.com/projects/g/ucbdrive/scalabel/rev/pr-b8242466c4ad9c6babef317150b2460d06672de2)

**new alerts:**

* 1 for Unused or undefined state property

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by fyu** [comment]
This pull request **introduces 1 alert** when merging 949771e184d0107d71d4646bc68c045893396bb8 into a14928b7bbbea2dbc6851b1a1a522ed30e1acdc3 - [view on LGTM.com](https://lgtm.com/projects/g/ucbdrive/scalabel/rev/pr-1a8882c7969f117a24fc6909363f238bd03826b2)

**new alerts:**

* 1 for Unused or undefined state property

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by fyu** [comment]
This pull request **introduces 1 alert** when merging 124149fcce81c9da280347940e9b22293483a138 into a14928b7bbbea2dbc6851b1a1a522ed30e1acdc3 - [view on LGTM.com](https://lgtm.com/projects/g/ucbdrive/scalabel/rev/pr-50cb2cf5a298f42490a7135f92bf5d0beef33ab6)

**new alerts:**

* 1 for Unused or undefined state property

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by fyu** [comment]
This pull request **introduces 1 alert** when merging c0dff7cd3a1ea300e1dc8c29b1122e7793cf0829 into a14928b7bbbea2dbc6851b1a1a522ed30e1acdc3 - [view on LGTM.com](https://lgtm.com/projects/g/ucbdrive/scalabel/rev/pr-b85a6e426c4e62c8a2491217e6652536ac32b311)

**new alerts:**

* 1 for Unused or undefined state property

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by fyu** [comment]
This pull request **introduces 1 alert** when merging 61511177cf97cd1d0d341424f57a6f251f8ba411 into a14928b7bbbea2dbc6851b1a1a522ed30e1acdc3 - [view on LGTM.com](https://lgtm.com/projects/g/ucbdrive/scalabel/rev/pr-4524037f2fe19762e0d379603f868bb728793536)

**new alerts:**

* 1 for Unused or undefined state property

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by fyu** [comment]
This pull request **introduces 2 alerts** when merging 16eb4c52a99e6613fccb82d0d690a9363fa35b6e into a14928b7bbbea2dbc6851b1a1a522ed30e1acdc3 - [view on LGTM.com](https://lgtm.com/projects/g/ucbdrive/scalabel/rev/pr-de3d229de95eb4d84e60c5016ce5e6a66849b552)

**new alerts:**

* 2 for Unused or undefined state property

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by fyu** [comment]
This pull request **introduces 1 alert** when merging c05d411efb36c74db05b6f008a94d6c1be2ec15c into a14928b7bbbea2dbc6851b1a1a522ed30e1acdc3 - [view on LGTM.com](https://lgtm.com/projects/g/ucbdrive/scalabel/rev/pr-b4dd2b5284452f0a44a6f4707649cbc1790565eb)

**new alerts:**

* 1 for Unused or undefined state property

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by fyu** [comment]
This pull request **introduces 1 alert** when merging 148e7e09c7f5da74ea6f4231785ceac78a43306a into a14928b7bbbea2dbc6851b1a1a522ed30e1acdc3 - [view on LGTM.com](https://lgtm.com/projects/g/ucbdrive/scalabel/rev/pr-95d56e78d5369cb3aaf2ade4102f7927f3671bbd)

**new alerts:**

* 1 for Unused or undefined state property

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by fyu** [comment]
This pull request **introduces 1 alert** when merging 33d389ad9d82408a10288e055d7790a1b44ac510 into a14928b7bbbea2dbc6851b1a1a522ed30e1acdc3 - [view on LGTM.com](https://lgtm.com/projects/g/ucbdrive/scalabel/rev/pr-ea8853ff8dea920a011af9d1f8f633992f46c76a)

**new alerts:**

* 1 for Unused or undefined state property

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by fyu** [comment]
This pull request **introduces 1 alert** when merging 016dbe04943e035ace98848aafa72e488538e336 into a14928b7bbbea2dbc6851b1a1a522ed30e1acdc3 - [view on LGTM.com](https://lgtm.com/projects/g/ucbdrive/scalabel/rev/pr-548566ea819baf35a988cd1a78ee84dfd34ce988)

**new alerts:**

* 1 for Unused or undefined state property

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by fyu** [comment]
This pull request **introduces 1 alert** when merging c805d0088aa3b7384971389f8ec609ff5c0b90f4 into a14928b7bbbea2dbc6851b1a1a522ed30e1acdc3 - [view on LGTM.com](https://lgtm.com/projects/g/ucbdrive/scalabel/rev/pr-bea76a754e9e58d8434d41727123cd21804c9f11)

**new alerts:**

* 1 for Unused or undefined state property

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by yuandong** [comment]
The config based setup looks good! Nitpicking on the styles for better usability: 
1. it will be easier to have a fixed width for each label category Item (maybe an optional attribute in the config?), and left align for each row (seems that current flexbox are central aligned)

Same for toolbar_switch, might be easier for the user if each item has the same width.

**Comment by fyu** [comment]
This pull request **introduces 1 alert** when merging 714d2cb804199e7c05ce219db9fb5e62f7abe5bd into a14928b7bbbea2dbc6851b1a1a522ed30e1acdc3 - [view on LGTM.com](https://lgtm.com/projects/g/ucbdrive/scalabel/rev/pr-f5b05552823513f4abbbe8f69dc182648c995989)

**new alerts:**

* 1 for Unused or undefined state property

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by fyu** [comment]
This pull request **introduces 1 alert** when merging b871dfc765272a2cc84c4635489b468ce42b6a5b into 093e3b2868429ac7db9a3889c885bdc6d85ad89e - [view on LGTM.com](https://lgtm.com/projects/g/ucbdrive/scalabel/rev/pr-66aeae51d8163cd63a4a97a489d7edb62cce94e6)

**new alerts:**

* 1 for Unused or undefined state property

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by fyu** [comment]
This pull request **introduces 1 alert** when merging 455947ea684324772fe52feec8d06071d54591b4 into 093e3b2868429ac7db9a3889c885bdc6d85ad89e - [view on LGTM.com](https://lgtm.com/projects/g/ucbdrive/scalabel/rev/pr-e8c209fb3bbd0d9bec4e909a1f94dcfd6919a7ff)

**new alerts:**

* 1 for Unused or undefined state property

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by fyu** [comment]
This pull request **introduces 2 alerts** when merging 132880b0d4209d0f43ba612f7fb5ecfe76dafd0a into 093e3b2868429ac7db9a3889c885bdc6d85ad89e - [view on LGTM.com](https://lgtm.com/projects/g/ucbdrive/scalabel/rev/pr-cbd243612f627bc32a421a133de5f2a996cb50af)

**new alerts:**

* 1 for Syntax error
* 1 for Unused or undefined state property

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by fyu** [comment]
This pull request **introduces 1 alert** when merging 732a41a51ef30c6a27a9d34475f94b750962614c into 093e3b2868429ac7db9a3889c885bdc6d85ad89e - [view on LGTM.com](https://lgtm.com/projects/g/ucbdrive/scalabel/rev/pr-494fc75722490f5e274fc4a3d443930e39c22bde)

**new alerts:**

* 1 for Unused or undefined state property

---

*Comment posted by [LGTM.com](https://lgtm.com)*

File: app/src/js/common/label.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+ * This is a Icon Button component that
+ * displays the remove.
+ * @param {object} props
+ * @return {jsx} component
+ */
+
+export function renderTemplate(toolType: any, handeleToogle: any, name: any, values: any) {
+    if (toolType === 'switch') {
+        return (
+            <SwitchBtn onChange = {handeleToogle} value = {name} />
```
Comment (yuandong) [inline app/src/js/common/label.tsx]: nit: handleToggle?

File: app/src/js/components/tool_bar.tsx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+import ListItem from '@material-ui/core/ListItem';
+import Divider from '@material-ui/core/Divider';
+import {renderTemplate, renderButtons} from '../common/label';
+import List from '@material-ui/core/List/List';
+import {genButton} from './general_button';
+
+interface Props {
+    categories: any[];
+    attributes: any[];
+    itemType: any;
```
Comment (yuandong) [inline app/src/js/components/tool_bar.tsx]: maybe use type literal? https://mariusschulz.com/blog/string-literal-types-in-typescript
Quite hard to see what type is supported

**Comment by fyu** [comment]
This pull request **introduces 1 alert** when merging dddf3e01b59e3a4e70d829c09396ee3645da2669 into 093e3b2868429ac7db9a3889c885bdc6d85ad89e - [view on LGTM.com](https://lgtm.com/projects/g/ucbdrive/scalabel/rev/pr-62b95d7de8fe96c4c54738199648b96f1d783469)

**new alerts:**

* 1 for Unused or undefined state property

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by fyu** [comment]
This pull request **introduces 1 alert** when merging f802ea44f7a5e65ba1ce236ac705e8d160632e54 into 093e3b2868429ac7db9a3889c885bdc6d85ad89e - [view on LGTM.com](https://lgtm.com/projects/g/ucbdrive/scalabel/rev/pr-8d629796d64193faf6a675a6cd3affce2b461516)

**new alerts:**

* 1 for Unused or undefined state property

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by fyu** [comment]
This pull request **introduces 1 alert** when merging 5ef60345c14b1a4baf3385bf7307fce938506cd1 into 093e3b2868429ac7db9a3889c885bdc6d85ad89e - [view on LGTM.com](https://lgtm.com/projects/g/ucbdrive/scalabel/rev/pr-bf2fc49ab02f96f8836910a133749386e1b974d6)

**new alerts:**

* 1 for Unused or undefined state property

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by fyu** [comment]
This pull request **introduces 1 alert** when merging df3fea598185df3f96e4035dbfac47bb3300cc52 into 093e3b2868429ac7db9a3889c885bdc6d85ad89e - [view on LGTM.com](https://lgtm.com/projects/g/ucbdrive/scalabel/rev/pr-8bab9b5c5dd3d9d0f40028caa4fe529359c749d5)

**new alerts:**

* 1 for Unused or undefined state property

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by fyu** [comment]
This pull request **introduces 1 alert** when merging c9487cd4dfe585c18b15b5da1a99bf8d58e13032 into 093e3b2868429ac7db9a3889c885bdc6d85ad89e - [view on LGTM.com](https://lgtm.com/projects/g/ucbdrive/scalabel/rev/pr-baf9572c25087af312ef2ed68bec85773408126b)

**new alerts:**

* 1 for Unused or undefined state property

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by fyu** [comment]
This pull request **introduces 1 alert** when merging 9614f17fcd781f850464799a6f19205ab39fdda2 into 093e3b2868429ac7db9a3889c885bdc6d85ad89e - [view on LGTM.com](https://lgtm.com/projects/g/ucbdrive/scalabel/rev/pr-c9c1ab1288e2cb4593ebefa7abfd33da088a4655)

**new alerts:**

* 1 for Unused or undefined state property

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by fyu** [comment]
This pull request **introduces 1 alert** when merging 7545cef921cc2a4f9f7a36c16896ab5d26d1cd99 into 093e3b2868429ac7db9a3889c885bdc6d85ad89e - [view on LGTM.com](https://lgtm.com/projects/g/ucbdrive/scalabel/rev/pr-af180ae4a6075393301566b6779a9fc6bb38a211)

**new alerts:**

* 1 for Unused or undefined state property

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by fyu** [comment]
This pull request **introduces 1 alert** when merging c79c87337e1853796718b7046185ab242347f374 into 093e3b2868429ac7db9a3889c885bdc6d85ad89e - [view on LGTM.com](https://lgtm.com/projects/g/ucbdrive/scalabel/rev/pr-8f4d5afb44a02dc336aa8a9ee228d8e107d00c64)

**new alerts:**

* 1 for Unused or undefined state property

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by fyu** [comment]
This pull request **introduces 1 alert** when merging 87f2e915089c2035186be063c568acfd8cc787db into 093e3b2868429ac7db9a3889c885bdc6d85ad89e - [view on LGTM.com](https://lgtm.com/projects/g/ucbdrive/scalabel/rev/pr-a9e09b11b4d40c2d0759ec4b660f331a989896ed)

**new alerts:**

* 1 for Unused or undefined state property

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by fyu** [comment]
This pull request **introduces 1 alert** when merging 00ee98f4135c8ad7df6bb74f8b15de321bbf3e6b into 093e3b2868429ac7db9a3889c885bdc6d85ad89e - [view on LGTM.com](https://lgtm.com/projects/g/ucbdrive/scalabel/rev/pr-4544bced17643b4106e36371482496451bc69598)

**new alerts:**

* 1 for Unused or undefined state property

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Pull request merged**